scripts/run_deploy_test.py

```diff
@@ -0,0 +1,1009 @@
+# Standard
+from typing import Dict, List
+import argparse
+import json
+import logging
+import os
+import sys
+import time
+
+# Third Party
+from kr8s.objects import Pod
+from testharness.utils.k8s.deployments import *
+
+# pip install caikit-test-harness --> ai-foundation, useful tools
+from testharness.utils.k8s.pods import *
+import grpc
+import kubernetes
+
+# requires running `python -m grpc_tools.protoc -I ./proto --python_out=. --grpc_python_out=. generation.proto` in TGIS repo
+import tgis.generation_pb2
+import tgis.generation_pb2_grpc
+import yaml
+
+# Default configurations to use for preset ConfigMap and tuning Pod
```

Comment:
I agree that using the twitter complaints dataset may be better since its smaller and faster. Wasn't sure for the quality metrics if we wanted to use the sentiment, tone, and entities dataset but those can be set by the user.